language_detector/main.py

```diff
@@ -1,8 +1,9 @@
-# -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counts = { lang['name']:0 for lang in languages }
+    words = text.split()
+    for w in words:
+        for lang in languages:
+            if w in lang['common_words']:
+                counts[lang['name']]+=1
+    return max((count, name) for name, count in counts.items())[1]
```

Comment:
OK, I understand now. The french speaking me got mixed up! But, in the meantime, I looked out for some shorter/clever solutions and I think this one is pretty nice:
```python
return max(counts, key=counts.get)
```
I can't totally wrap my head around it, but it's nice.